src/poliastro/tests/test_czml.py

```diff
@@ -1,19 +1,11 @@
 import pytest  # noqa: E402 isort:skip
-import sys
 
 from astropy import units as u  # noqa: E402
 
+from poliastro.czml.extract_czml import CZMLExtractor  # noqa: E402
 from poliastro.examples import iss, molniya  # noqa: E402
 
-try:
-    from poliastro.czml.extract_czml import CZMLExtractor  # noqa: E402
 
-    import czml3  # noqa: F401
-except ImportError:
-    pass
-
-
-@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")
```

Comment:
Let's leave this like it was before, now that we are properly installing it in the CI